### DISCUSSION TITLE
Transaction Date

### DISCUSSION THREAD

**Discussion by SlightlyFixated** [firefly-iii/firefly-iii#9535] [Firefly III] [answered]
What is the correct Role to set to have the transaction date from my CSV file used. I've used Transaction Date and all of the transaction dates are set to "todays date". I've tried the other Transaction Date roles (posted, due, etc) and can't seem to get this to work.

Thanks!

**Comment by JC5** [comment]
Hey, thanks for starting a discussion.

Did you use the correct date format?

**Comment by JC5** [comment, marked as answer]
See also: https://docs.firefly-iii.org/references/faq/data-importer/import/#all-transactions-are-imported-as-today

**Reply by SlightlyFixated** [reply]
Thank you, that worked. I will read the documentation more thoroughly before posting, next time. 👍
